src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -204,7 +204,7 @@ studio.configuration.management.previewProtectedUrls: >-
   /api/1/site/cache/statistics
 
 # The pattern used to load plugins installed from the marketplaces
-studio.configuration.plugin.base.pattern: /config/studio/plugins/${id}/${type}/${name}
+studio.configuration.plugin.base.pattern: /config/studio/plugins/js/${id}/${type}/${name}
```

Comment:
Is this a fix to a prior bug?

Reply:
Yes, it was introduced with the PR for installing plugins, it's new code that none has used yet